birdhouse/README.rst

```diff
@@ -0,0 +1,259 @@
+.. role:: raw-html-m2r(raw)
```

Comment:
Just curious, what is this `role:: raw-html-m2r(raw)`?

Reply:
`m2r` is a library for converting markdown to rst. If we need to push some fixes, I'll address this.